code/datums/gamemode/factions/bloodcult/friendlycult/friendlycult_runes.dm

```diff
@@ -0,0 +1,414 @@
+
+
+
+
+var/list/friendly_rune_list = list()//all peaceful runes currently in the world
+
+/obj/effect/friendly_rune
```

Comment:
Is this entire file an exact copypaste of regular cult runes?

Reply:
Yes. I will be working on altering the system to better allow modular runesets soon (tm).